modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -340,7 +340,7 @@ State:                      ready
 Private:                    No
 Created:                    Oct  1 2021 08:12:25 UTC
 Details Page:               https://console.redhat.com/openshift/details/s/<subscription_id>
-OIDC Endpoint URL:          https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id>
```

Comment:
**Peer Review**: This `<CloudFront_generated_hash>` will be replaced shortly with the proper subdomain.